### ISSUE TITLE
45.3 Overview ->Parts in Use sub-tab displays nothing.

### ISSUE THREAD

**Issue by Apocal43** [MegaMek/mekhq#1139]
MekHQ 45.3
Windows 10
Java version 1.8.0_201

Under the "Overview" tab, the "Parts in Use" sub-tab displays nothing. 
![image](https://user-images.githubusercontent.com/43554459/53750331-705b2d80-3ee4-11e9-8d41-48856d72e9c2.png)

[mekhqlog.txt](https://github.com/MegaMek/mekhq/files/2927456/mekhqlog.txt)

[Phantom Frost Supernova Trinary20.zip](https://github.com/MegaMek/mekhq/files/2927459/Phantom.Frost.Supernova.Trinary20.zip)

**Comment by VicenteCartas** [comment]
I think this is a duplicate bug fixed by #1011

**Comment by Wrathbow** [comment]
I have had the same issue in 0.44.0 and now in 0.45.4
I have been using MekHQ to track a role-play campaign that I am running for my friends.  When this previously happened, I downloaded the newest version of the program, and painstakingly recreated the entire (now over 1 year old in universe, 6 months in teal time) campaign.  The tab worked fine until the most recent one or two saves.
I am not using any power armor or exoskeletons, and the system did not balk at custom 'Mechs or modifying the main DropShip.  I am uncertain what changed.
The campaign is listed under Nemo. Aside from my custom units (in a folder labeled Customs), the only changes I have made is to reorganizing and adding camo, and reorganizing and adding awards  This was done long ago, and seemed to cause no issues at the time, so I find it unlikely that the issue is from this point.
As listing #1010 appears to deal with implementation of battle armor, and not the inventory list, I am unsure how that bug correction could correct/close this one, but as I am not a programmer, I wouldn't know.  However, as the problem seems to still persist into a new release, it suggests that something is still wrong.

My attempt to upload the files here note the file as being too big.  If I know which portions to upload, I will gladly do so.
